_posts/2018-11-29-craft-consultants.md

```diff
@@ -12,9 +12,9 @@ square_related:
 feature_image: posts/2018-11-29-craft-consultants/lego-superheroes.jpg
 ---
 
-> As aspiring Software Craftsmen we are raising the bar of professional software development by practicing it and helping others learn the craft.
+> As aspiring Software Crafters we are raising the bar of professional software development by practicing it and helping others learn the craft.
```

Comment:
As for the quotation, what I have learned is that it can be changed like so:
```
> As aspiring Software [Crafters] we are raising the bar of professional software development by practicing it and helping others learn the craft.
```

Source: https://www.chicagomanualofstyle.org/home.html